src/NServiceBus.AcceptanceTesting/AcceptanceTestingTransport/AcceptanceTestingTransport.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NServiceBus
+{
+    using Routing;
+    using Settings;
+    using Transport;
+
+    public class AcceptanceTestingTransport : TransportDefinition, IMessageDrivenSubscriptionTransport
+    {
+        public override bool RequiresConnectionString => false;
+
+        public override string ExampleConnectionStringForErrorMessage { get; } = "";
+
+        public override TransportInfrastructure Initialize(SettingsHolder settings, string connectionString)
+        {
+            Guard.AgainstNull(nameof(settings), settings);
+
+            var useNativePubSub = settings.GetOrDefault<bool>("AcceptanceTestingTransport.UseNativePubSub");
```

Comment:
Didn't we agree to invert this? (since most downstreams would like the native pubsub and timeouts to be used)

Reply:
yes, I inverted it from the previous behavior...happy to change it back again